deploy/grafana/grafana-with-grafana-operator/install-grafana-with-operator.sh

```diff
@@ -77,7 +77,7 @@ kubectl apply --namespace="${GRAFANA_NAMESPACE}" -f <( \
     envsubst \
 )
 
-echo "Waiting to apply GraganaDatasource custom resource $GRAFANA_PROMETHEUS_DATASOURCE_NAME"
+echo "Waiting to apply Grafana Datasource custom resource $GRAFANA_PROMETHEUS_DATASOURCE_NAME"
```

Comment:
grafana operator implements `GrafanaDataSource` custom resource ;)
look https://github.com/integr8ly/grafana-operator/blob/master/documentation/datasources.md
but doesn't matter
thanks, you for your efforts @wiktor2200

Reply:
Typo was Gra**g**ana instead of Gra**f**ana, I can see, that space was unnecessary ;)